docs/index.rst

```diff
@@ -88,6 +88,7 @@ Full Contents
 .. toctree::
    :maxdepth: 2
  
+   contributing
```

Comment:
Perhaps this should be underneath "install"? I'd assume someone would first want to use this in the context of a JupyterHub before they'd consider installing it locally. I think that your points about the extension working on a "regular" Jupyter server are also relevant to someone who is not contributing, so perhaps all the "generally useful, not just for contributing" stuff could be moved [to the installation page](https://jupyterhub.github.io/nbgitpuller/install.html#)?

Reply:
Moved it around. Running it just on your local machine without a hub is probably not common enough, so let's leave it in contributing for now.

Reply:
sounds good - I don't think it's worth blocking the PR on the final point, though I guess the question is whether we assume most users of nbgitpuller will start with a hub, *then* want to investigate nbgitpuller, or whether they'll wonder about nbgitpuller before their hub exists. If it is the latter, then it is pretty cumbersome to tell someone "you can't learn about how nbgitpuller feels when you use it until you first set up a JupyterHub".